src/utils/serverRequests.js

```diff
@@ -64,4 +64,53 @@ export const updateItemQuantity = async(id, dispatch, updateType) => {
     }catch(err){
       console.log(err);
     }
+}
+
+export const getWishlist = async(dispatch) => {
+  try{
+    const {status, data} = await axios({
+      method: "get",
+      url: "/api/user/wishlist",
+      headers: {authorization: localStorage.getItem("encodedToken")}
+    });
+    console.log("Calling api get wishlist");
```

Comment:
Is this required now?

Reply:
I have added it to keep a check on the API calls. Will remove it.